fmn/consumer/backends/mail.py

```diff
@@ -148,7 +148,8 @@ def _format_line(msg):
 
         n = len(queued_messages)
         subject = u"Fedora Notifications Digest (%i updates)" % n
-        content = "\n".join([
+        separator = "\n\n" + "-"*80 + "\n\n"
```

Comment:
Looks good to me.. can I ask that we reduce this to 79 ``-`` chars?  Sometimes 80 is just one too many for some people.